lib/cache/cache.go

```diff
@@ -1264,3 +1264,13 @@ func (c *Cache) GetAuthPreference() (services.AuthPreference, error) {
 	defer rg.Release()
 	return rg.clusterConfig.GetAuthPreference()
 }
+
+// GetPAMConfig gets the cluster PAM config.
+func (c *Cache) GetPAMConfig() (types.PAMConfig, error) {
+	rg, err := c.read()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	defer rg.Release()
+	return rg.clusterConfig.GetPAMConfig()
+}
```

Comment:
This resource also needs to be added to the caches for the services that will need it.

https://github.com/gravitational/teleport/blob/833e9a351bd6d8e9bb474b53ec14934e342d6829/lib/cache/cache.go#L42-L71

You might want to take a look at `KindDatabase` with `git blame` to see how it was added and propagated everywhere because you have to updated it in `lib/services/local/events.go` and `lib/auth/grpcserver.go` and `lib/cache/collections.go`.

Reply:
@fspmarshall Can walk you through this.

Reply:
should be all fixed up now and ready for a second pass